bpf/Makefile

```diff
@@ -112,12 +123,18 @@ load-map-cgroup_info_map:
 load-map-mark_pod_ips_map:
 	[ -f $(PROG_MOUNT_PATH)/mark_pod_ips_map ] || sudo bpftool map create $(PROG_MOUNT_PATH)/mark_pod_ips_map type hash key 4 value 16 entries 65535 name mark_pod_ips_map
 
+load-map-settings:
+	[ -f $(PROG_MOUNT_PATH)/settings ] || sudo bpftool map create $(PROG_MOUNT_PATH)/settings type hash key 4 value 16 entries 256 name settings
+
 load-map-pair_original_dst:
 	[ -f $(PROG_MOUNT_PATH)/pair_original_dst ] || sudo bpftool map create $(PROG_MOUNT_PATH)/pair_original_dst type lru_hash key 36 value 24 entries 65535 name pair_original_dst
 
 load-map-sock_pair_map:
 	[ -f $(PROG_MOUNT_PATH)/sock_pair_map ] || sudo bpftool map create $(PROG_MOUNT_PATH)/sock_pair_map type sockhash key 36 value 4 entries 65535 name sock_pair_map
 
+load-map-process_events:
+	[ -f $(PROG_MOUNT_PATH)/process_events ] || sudo bpftool map create $(PROG_MOUNT_PATH)/process_events type perf_event_array key 4 value 4 entries $(shell nproc) name process_events
+
```

Comment:
Good catch.

Reply:
"loadall" will allow two eBPF progs to share one map, and since we don't need to operate this map from the control plane, we actually don't need to manually pin this map.